admin/class-admin-init.php

```diff
@@ -59,7 +59,10 @@ public function enqueue_dismissible() {
 	 * Redirect first time or just upgraded users to the about screen.
 	 */
 	public function after_update_notice() {
-		if ( current_user_can( 'manage_options' ) && ! $this->seen_about() ) {
+
+		$can_access = is_multisite() ? WPSEO_Utils::grant_access() : current_user_can( 'manage_options' );
```

Comment:
It seems to me only superadmins/admins should be seeing this notice. `grant_access` has a pretty nasty implementation if you ask me and it doesn't really state quite clearly in documentation why it does what it does. `grant_access()` will always return true if `wpseo_ms` option is not set. To be sure of right capability here you'd need to rewrite to: 
```
$can_access = is_multisite() ? WPSEO_Utils::grant_access() && current_user_can( 'manage_options' ) : current_user_can( 'manage_options' );
```